.circleci/config.yml

```diff
@@ -30,60 +34,99 @@ common_filters: &common_filters
       ignore:
         - gh-pages
 
+common_jobs: &common_jobs
+    jobs:
+        - find-develop-snapshot
+        - build:
+            <<: *common_filters
+            requires:
+              - find-develop-snapshot
+        - unit-tests:
+            <<: *common_filters
+            requires:
+              - build
+        - integration-tests:
+            matrix:
+              parameters:
+                testing_profile: [ "singularity-tests", "bitbucket-tests" ]
+            <<: *common_filters
+            requires:
+              - build
+        - non-confidential-tests:
+            <<: *common_filters
+            requires:
+              - build
+        - confidential-workflow-tests:
+            <<: *common_filters
+            requires:
+              - build
+        - confidential-tool-tests:
+            <<: *common_filters
+            requires:
+              - build
+        - wes-toil-test:
+            <<: *common_filters
+            requires:
+              - build
+        - sonar-cloud:
```

Comment:
I was torn on whether we should run SonarCloud on the nightly build against develop, but decided to leave it. Who knows, there might be some weird edge case where it reveals something.